@xen-orchestra/lite/src/views/new-vm/NewVmView.vue

```diff
@@ -523,6 +541,18 @@ const addStorageEntry = () => {
 //   vmState.sshKeys.splice(index, 1)
 // }
 
+const addTag = () => {
+  const tag = vmState.tag.trim()
```

Comment:
You can retrieve your tag directly via the HTML event, so you won't need the "tag" property anymore.
```suggestion
const addTag = (ev: InputEvent & { target: HTMLInputElement }) => {
  const tag = ev.target.value.trim()
```

Reply:
As discussed on MM, I can keep the "tag" property until the component is changed